packages/react-server/src/ReactFizzHooks.js

```diff
@@ -22,6 +22,8 @@ import type {
 import type {ResponseState} from './ReactFizzConfig';
 import type {Task} from './ReactFizzServer';
 import type {ThenableState} from './ReactFizzThenable';
+// $FlowFixMe - Flow complains that this module doesn't exist, but only for some configs. No idea why.
+import type {FormStatus} from 'react-dom-bindings/src/shared/ReactDOMFormActions';
```

Comment:
If I remove this Flow suppression comment, I get a really weird error:


```
Running Flow on the native-relay renderer...
Error ---------------------------------------------------------------- packages/react-server/src/ReactFizzHooks.js:25:31

Cannot resolve module `react-dom-bindings/src/shared/ReactDOMFormActions`. [cannot-resolve-module]

   25| import type {FormStatus} from 'react-dom-bindings/src/shared/ReactDOMFormActions';
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


Error ---------------------------------------------------------------- packages/react-server/src/ReactFizzHooks.js:46:26

Cannot resolve module `react-dom-bindings/src/shared/ReactDOMFormActions`. [cannot-resolve-module]

   46| import {NotPending} from 'react-dom-bindings/src/shared/ReactDOMFormActions';
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

What's especially weird is that it only fails when running the Flow config for a non-DOM renderer. The DOM config works fine. I snooped around our Flow infra and couldn't find any reason this would matter.

I also tried moving ReactDOMFormActions back into react-dom, but no luck. Really perplexing.

Here's the failed job: https://app.circleci.com/pipelines/github/facebook/react/41790/workflows/85df201a-829b-4ff5-92af-a7ed3e9e74f2/jobs/677693/parallel-runs/1?filterBy=FAILED